### PR TITLE
Custom weekday builder.

### PR DESCRIPTION
**Pull request by maxgmer**
A custom builder for weekdays.

### CODE REVIEW

**Comment by maxgmer** [comment]
Still relevant, writing to draw attention)

**Comment by aleksanderwozniak** [comment]
Heh yes, I am aware, I didn't have much time lately.

File: lib/src/customization/calendar_builders.dart [hunk 1]
```diff
@@ -61,6 +65,10 @@ class CalendarBuilders {
   /// Mutually exclusive with `markersBuilder`.
   final SingleMarkerBuilder singleMarkerBuilder;
 
+  /// Custom builder for weekday names. They are displayed below `HeaderRow` and above calendar days.
+  /// Will overwrite `weekdayStyle` and `weekendStyle` from `DaysOfWeekStyle`.
+  final WeekdayBuilder weekdayBuilder;
```
Comment (aleksanderwozniak) [inline lib/src/customization/calendar_builders.dart]: Lets name this `dowBuilder` to keep consistent naming.

I think it also might be beneficial to separate week days from weekends. Can you add a second builder?
Naming could be then as follows: `dowWeekdayBuilder` and `dowWeekendBuilder`.

Comment (maxgmer) [reply lib/src/customization/calendar_builders.dart]: Just want to clarify, if I got it right:
final WeekdayBuilder weekdayBuilder; => final DowBuilder dowWeekdayBuilder;

Also add another builder for 2 weekend days from weekday row, which will override dowWeekdayBuilder for weekends, if both exist:
final DowBuilder weekendBuilder;
...

**Pull request merged**